libopenage/gamestate/player.cpp

```diff
@@ -120,19 +128,31 @@ void Player::active_unit_added(Unit *unit) {
 		return;
 	}
 
+	this->units_have[unit->unit_type->id()] += 1;
+	this->units_had[unit->unit_type->id()] += 1;
```

Comment:
The default initializer of the int gets invoked, so it's guaranteed to be 0. Its not like the:
```
int i;
```
Its like
```
int i = int();
```